interface/main/calendar/includes/pnMod.php

```diff
@@ -700,6 +700,8 @@ function pnModURL($modname, $type = 'user', $func = 'main', $args = array(), $pa
     //this may break windows path accesses?
     $path = preg_replace("/[^\.\/a-zA-Z0-9]/", "", $path)
     ;
+    //convert url characters html entities
+    $url=attr($url);
     // The URL
```

Comment:
In the calendar template there is  a lot of urls that builded with this function and it unsecured.

examples -
openemr/interface/main/calendar/index.php
![image](https://user-images.githubusercontent.com/17809866/30636459-2f002666-9dfe-11e7-94d3-29c8f920baf2.png)

![image](https://user-images.githubusercontent.com/17809866/30636526-66725cd6-9dfe-11e7-8cca-7a901de566da.png)

Reply:
Agree with this change also.